File: js/schedule-expiration/UpdateModel.js
// DocSection: schedule_expiration_update_model
export class LandingPage extends KontentDelivery.ContentItem {
    constructor(){
         super({
             propertyResolver: ((elementName) => {
                 if (elementName === 'expire_at'){
                     return 'ExpireAt';
                 }
               
                 ...
             }),
         })
     }
 }
// EndDocSection

File: java/schedule-expiration/UpdateModel.java
// DocSection: schedule_unpublishing_update_model
@ElementMapping("expire_at")
ZonedDateTime ExpireAt;

public ZonedDateTime getExpireAt() {
  	return ExpireAt;
}

public void setExpireAt(ZonedDateTime ExpireAt) {
  	this.ExpireAt = ExpireAt;
}
// EndDocSection